matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/aggregation/livelocation/DefaultLiveLocationAggregationProcessor.kt

```diff
@@ -83,6 +85,7 @@ internal class DefaultLiveLocationAggregationProcessor @Inject constructor() : L
                 ?: 0
 
         if (updatedLocationTimestamp.isMoreRecentThan(currentLocationTimestamp)) {
+            Timber.d("set last location of summary with id=$targetEventId to ${content.getBestLocationInfo()}")
```

Comment:
We should not log the location of the user for privacy reason

Reply:
Okay I didn't know. I will remove the location info then.

Reply:
Removed. See [a971b19](https://github.com/vector-im/element-android/pull/5869/commits/a971b19f5ce511cd307f66f6d3202d3ad5341200)